cli/command/service/opts_test.go

```diff
@@ -250,3 +251,47 @@ func TestHealthCheckOptionsToHealthConfigConflict(t *testing.T) {
 	_, err := opt.toHealthConfig()
 	assert.Error(t, err, "--no-healthcheck conflicts with --health-* options")
 }
+
+func TestSecretOptionsSimple(t *testing.T) {
+	var opt SecretOpt
+
+	testCase := "source=/foo,target=testing"
```

Comment:
Hum, `/foo` is an invalid secret name right ? It might confuse people reading the test (to see the `/foo` syntax) 👼